Sming/Makefile

```diff
@@ -554,14 +554,15 @@ dist-clean: clean samples-clean third-party-clean libraries-clean tools-clean
  	done
 	$(Q) $(GIT) checkout $(USER_LIBDIR)
 
+# Recursive wildcard search
+rwildcard = $(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2) $(filter $(subst *,%,$2),$d))
 
 # Files that should follow our coding standards
-CS_FILES=$(shell find SmingCore/ -name '*.cpp' -o  -name '*.h' -o -name '*.c') \
-         $(shell find ../samples -name '*.cpp' -o  -name '*.h' -o -name '*.c')
+CS_FILES := $(call rwildcard, SmingCore ../samples, *.cpp *.h *.c)
 
 cs:
 	$(Q) for FILE in $(CS_FILES); do \
-		clang-format -i -style=file $$FILE; \
+		clang-format -verbose -i -style=file $$FILE; \
```

Comment:
The `-verbose` switch is not present. At least not in `clang-format version 5.0.0-3~16.04.1 (tags/RELEASE_500/final)` for Linux. Please, remove it too.

Reply:
> OK, without -verbose though I do think some sort of message is required to confirm it's working. 

Well the tool will do its job and if there is something changed it can be be seen when the developer types: `git status` or `git diff`. A bit more verbosity can be achieved directly with
`make cs V=1`. So just removing the `-verbose` switch for this PR will be enough.